opm/autodiff/ParallelOverlappingILU0.hpp

```diff
@@ -193,12 +371,57 @@ class ParallelOverlappingILU0
         DUNE_UNUSED_PARAMETER(x);
     }
 
-private:
+protected:
+    void init( const Matrix& A, const int iluIteration )
+    {
+        int ilu_setup_successful = 1;
+        std::string message;
+        const int rank = ( comm_ ) ? comm_->communicator().rank() : 0;
+
+        std::unique_ptr< Matrix > ILU;
```

Comment:
For ILU0 you make a copy and for ILUn the matrix is constructed useing row_wise, which btw is buggy, because the nonzeroes are not set. There is a fix in newer versions of dune-istl.